src/OpenTelemetry/README.md

```diff
@@ -5,17 +5,19 @@
 
 * [Installation](#installation)
 * [Introduction](#introduction)
-* [Getting started with Logs](#getting-started-with-logging)
-* [Getting started with Traces](#getting-started-with-tracing)
-* [Tracing Configuration](#tracing-configuration)
-  * [ActivitySource](#activity-source)
+* [Getting started with Logging](#getting-started-with-logging)
+* [Getting started with Tracing](#getting-started-with-tracing)
+* [Tracing configuration](#tracing-configuration)
+  * [Activity Source](#activity-source)
   * [Instrumentation](#instrumentation)
   * [Processor](#processor)
   * [Resource](#resource)
   * [Sampler](#sampler)
 * [Advanced topics](#advanced-topics)
   * [Propagators](#propagators)
 * [Troubleshooting](#troubleshooting)
+  * [Configuration Parameters](#configuration-parameters)
+  * [Remarks](#remarks)
```

Comment:
Both `Configuration Parameters` and `Remarks` seem to be too detailed and generic for the ToC. Perhaps we want to remove the H3 (e.g. replace `### Remarks` with `**Remarks:**`).

Reply:
@open-telemetry/dotnet-maintainers WDYT - should I change it?

BTW I think that the `Propagators` section should be under `Tracing configuration` - and not under `Advanced topics`.

Reply:
I will make a separate PR for it as I did not get feedback on what is the preference